superset/views/database/api.py

```diff
@@ -113,12 +114,20 @@ class DatabaseRestApi(DatabaseMixin, BaseSupersetModelRestApi):
     datamodel = SQLAInterface(Database)
 
     include_route_methods = {"get_list", "table_metadata", "select_star"}
-    class_permission_name = "DatabaseView"
-    method_permission_name = {
+
```

Comment:
@mistercrunch Had a doubt, Was wondering if this should be bundled in with the permission "Database"